src/views/UserProfile.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="user-profile">
+    <h1>This is the profile view</h1>
+    <div class="content">
+       <ShowNotes></ShowNotes>
+        <button class="logout-button" v-on:click="logOut">Logout</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import ShowNotes from '@/components/show-notes.vue'
+import { clearSession } from '@/user.js'
+
+export default {
+  name: 'userprofile',
+  components: {
+    ShowNotes
+  },
+  methods: {
+    logOut: function () {
+      clearSession()
```

Comment:
this method return a promise. u should handle the success request callback to delete jwt and moving to root route.